cmake/machinecfg/CI.cmake

```diff
@@ -0,0 +1,36 @@
+#========================================================================================
```

Comment:
Should we bother making a generic "mac" or "linux" machine file? Or perhaps the CI file is similar enough?

It would be good if one of the people using power9s on Darwin, say @AndrewGaspar , @jdolence , or @jlippuner added a file for that one.

Reply:
I think that a generic Mac file would be great (based on what I've heard about building programs on MacOS, e.g., with GCC or OpenMP).

Also, in addition to power9 on Darwin I got the impression that RZAnsel (?) is also used quite a bit by many people involved in development here.
May be worth adding a file for that machine, too.